ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -267,16 +280,7 @@ open class NavigationBar: UINavigationBar {
     private let trailingSpacerView = UIView() //defines the trailing space between the left and right barbuttonitems stack
     private var topAccessoryView: UIView?
     private var topAccessoryViewConstraints: [NSLayoutConstraint] = []
-
-    private var showsLargeTitle: Bool = true {
-        didSet {
-            if showsLargeTitle == oldValue {
-                return
-            }
-            updateAccessibilityElements()
```

Comment:
no consequences on accessibility updates by deleting these lines?

Reply:
No consequences, no. showsLargeTitle is a private var that is only set in one place. updateAccessibilityElements() now gets called in that other place.